src/lib/utils.ts

```diff
@@ -60,10 +60,10 @@ function getAvailabilityProps(
 
 function getSupplyPriorityProps(priority: SupplyPriority) {
   switch (priority) {
-    case SupplyPriority.UnderControl:
+    case SupplyPriority.NotNeeded:
       return {
-        label: 'Sob controle',
-        className: 'bg-light-yellow',
+        label: 'Não preciso',
```

Comment:
O mesmo aqui nesse label
```suggestion
        label: 'Não precisa',
```